Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/LtpValidationCondition.java

```diff
@@ -0,0 +1,187 @@
+     * (e.g., equal, not_equal, smaller_than, etc.)
+     * 
+     * @return the operation that is used to check the property against the condition values
+     */
+    @Override
+    public LtpValidationConditionOperation getOperation() {
+        return operation;
+    }
+
+    /**
+     * Return a list of values that are checked against the property of the file.
+     * 
+     * <p>May be a single value (comparing via equal), two values (comparing as interval) or 
+     * multiple values (comparing as set) depending on the operation</p>
+     * 
+     * @return the list of values
+     */
+    @Override
+    public List<String> getValues() {
+        return values;
+    }
+
+    /**
+     * Return the severity of the validation condition, whether the condition is critical
+     * and should be treated as an error, or whether the condition is optional and should
+     * be treated as a warning.
+     * 
+     * @return the severity of the validation condition
+     */
+    @Override
+    public LtpValidationConditionSeverity getSeverity() {
+        return severity;
+    }
+
+    /**
+     * Return the LTP validation configuration this condition belongs to.
+     * 
+     * @return the LTP validation configuration this condition belongs to
+     */
+    public LtpValidationConfiguration getLtpValidationConfiguration() {
+        return ltpValidationConfiguration;
+    }
+
+    /**
+     * Sets the property of the file that is checked.
+     * 
+     * @param property the property that is checked
+     */
+    public void setProperty(String property) {
+        this.property = property;
+    }
+
+    /**
+     * Sets the condition operation that is applied when checking the condition.
+     * 
+     * @param operation the condition operation that is applied
+     */
+    public void setOperation(LtpValidationConditionOperation operation) {
+        this.operation = operation;
+    }
+
+    /**
+     * Sets the values that are used for comparison when checking the condition.
+     * 
+     * @param values the list of values that are used for comparison
+     */
+    public void setValues(List<String> values) {
+        this.values = values;
+    }
+
+    /**
+     * Sets the failure severity that is used in case the condition does not pass.
+     * 
+     * @param severity the failure severity
+     */
+    public void setSeverity(LtpValidationConditionSeverity severity) {
+        this.severity = severity;
+    }
+
+    /**
+     * Sets the LTP validation configuration this condition belongs to.
+     * 
+     * @param ltpValidationConfiguration the LTP validation configuration this condition belongs to
+     */
+    public void setLtpValidationConfiguration(LtpValidationConfiguration ltpValidationConfiguration) {
+        this.ltpValidationConfiguration = ltpValidationConfiguration;
+    }
+
+    /**
+     * Equals implementation based on the database id.
+     */
```

Comment:
I find the comment valuable, but I would write it as a non-Javadoc comment `/*` instead of `/**`.
```suggestion
    /*
     * Equals implementation based on the database ID.
     */
```